config/migrations/20240111000000-enrich-skos-concept-BestuursorgaanClassificatieCode.sparql

```diff
@@ -0,0 +1,17 @@
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX conceptscheme: <http://data.vlaanderen.be/id/conceptscheme/>
+PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
+
+INSERT {
+  GRAPH ?h {
+    ?s a skos:Concept;
+      skos:inScheme conceptscheme:BestuursorgaanClassificatieCode;
+      skos:topConceptOf conceptscheme:BestuursorgaanClassificatieCode.
+  }
+}
+WHERE {
+  GRAPH ?g {
+    ?s a ext:BestuursorgaanClassificatieCode.
+    BIND(?g as ?h). # Workaround to avoid Virtuoso SR002 22023 error
```

Comment:
wow that is terrible. someone must have spent a long time figuring this one out :see_no_evil: